rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml

```diff
@@ -141,4 +141,4 @@ secure_cluster_flags:
 
 # We need to override the OpenStack upper constraint for Mitaka which is version 1.9.0, We should be able to remove this in Newton as the upper constraint moves to 2.3.0
 repo_build_upper_constraints_overrides:
-  - "elasticsearch==2.3.0"
+  - "elasticsearch===2.3.0"
```

Comment:
well, it now matches the syntax requirements lays down, not sure if strictly needed, but more of a correctness fix